__tests__/util_promiseCallbackInterop.coffee

```diff
@@ -0,0 +1,125 @@
+{ promiseCallbackInterop } = require '../lib/coffeescript/util'
+testResult = { results: ['a result'] }
+testError = { error: 'messed up' }
+testFun1 = (callback) ->
+    setTimeout(->
+        callback(undefined, testResult)
+    )
+testFun1Error = (callback) ->
+    setTimeout(->
```

Comment:
Why paranthesis? I would do:

setTimeout ->
    callback undefined, testResult

    return

I think it's important we follow the style of the code in the project instead of having multiple code styles.